xml/System.Diagnostics/ProcessStartInfo.xml

```diff
@@ -229,9 +229,11 @@
         <ReturnType>System.Collections.ObjectModel.Collection&lt;System.String&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a collection of command-line arguments to use when starting the application.</summary>
+        <value>A collection of command-line arguments.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[If you use this property to set command-line arguments, <xref:System.Diagnostics.ProcessStartInfo.Arguments> must be set to an empty string.]]></format>
```

Comment:
Since the property is read-only and there's no constructor that populates the ArgumentList collection, it would be helpful to explain how to populate the collection.